bundles/org.openhab.binding.millheat/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+  xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+  <config-description uri="thing-type:millheat:account">
+    <parameter name="username" type="text">
+      <label>Username</label>
+      <description>Your Millheat app username (email)</description>
+      <required>true</required>
+    </parameter>
+
+    <parameter name="password" type="text">
+      <label>Password</label>
+      <description>Your Millheat app password</description>
+      <required>true</required>
+      <context>password</context>
+    </parameter>
+
+    <parameter name="refreshInterval" type="integer" required="false" min="30" unit="s">
+      <label>Refresh interval</label>
+      <description>Specifies the refresh time in seconds for polling temperature updates from Millheat service</description>
+      <default>120</default>
+    </parameter>
+
+  </config-description>
+
+  <config-description uri="thing-type:millheat:heater">
+    <parameter name="macAddress" type="text" required="false">
+      <label>MAC address</label>
+    </parameter>
+    <parameter name="heaterId" type="text" required="false">
+      <label>Heater ID - use for independent heaters</label>
```

Comment:
Probably should be in description: `- use for independent heaters`

Reply:
Moved